src/pages/index.svelte

```diff
@@ -3,27 +3,40 @@
     import { flip } from 'svelte/animate'
 	import Player from '../components/player.svelte'
 	import AddPlayer from '../components/addPlayer.svelte'
-	import Button from '@smui/button'
+	import Button , {Label} from '@smui/button'
 	import Card from '@smui/card'
-	import Paper, {Title, Subtitle, Content} from '@smui/paper'
```

Comment:
Was unsure if I would used these in Paper at a later time, but they were conflicting with dialog.

Reply:
good call, bring them in and deal with the issues at that time.